company/careers.md

```diff
@@ -0,0 +1,54 @@
+# Careers
+
+Join us on our mission to make it so everyone, in every community, in every country, and in every industry can create products using the best technology. Read our [Master Plan](https://about.sourcegraph.com/plan) to learn more about what we're building, and why it matters.
+
+## Open positions
+
+We're hiring! Check out our open positions:
+
+### Engineering
+
+- [Engineering Manager - Web](../handbook/engineering/hiring/engineering-manager-web.md)
+- [Software Engineer - Frontend](../handbook/engineering/hiring/software-engineer-frontend.md)
+- [Software Engineer - Backend](../handbook/engineering/hiring/software-engineer-backend.md)
+- [Software Engineer - Distribution](../handbook/engineering/hiring/software-engineer-distribution.md)
+- [Software Engineer - Code Intelligence](../handbook/engineering/hiring/software-engineer-code-intelligence.md)
+- [Software Engineer - Security](../handbook/engineering/hiring/software-engineer-security.md)
+- [Software Engineer - Full stack](../handbook/engineering/hiring/software-engineer-full-stack.md)
+
+### Product
+
+- [Product Manager](../handbook/product/roles/product_manager.md)
+- [Technical Writer](../handbook/product/roles/product_manager.md)
```

Comment:
Would not include the Technical Writer role as we're not taking additional applications.

Reply:
```suggestion
```